lib/elixir_google_scraper_web/templates/keyword/html_response.html.eex

```diff
@@ -0,0 +1,5 @@
+<%= if @keyword.status == :completed do %>
+  <iframe srcdoc="<%= @keyword.search_result.html %>"></iframe>
+<% else %>
+  <h4 class="text-center">No search reasult for: <%= @keyword.name %></h4>
```

Comment:
```suggestion
  <h4 class="text-center">No search result for: <%= @keyword.name %></h4>
```

Same for https://github.com/junan/elixir_google_scraper/pull/42/files#diff-486ba74bd7d41669927b7b8e5c2e4fc7f3d50556f993a7cc7a476e73906d9909R17